quilt/tools/build.py

```diff
@@ -85,11 +89,14 @@ def build_package(username, package, yaml_path):
         raise BuildException("Unable to parse YAML: %s" % yaml_path)
 
     tables = data.get('tables')
```

Comment:
Since we're going to allow packages with no tables (or files), I'd do this:
```
tables = data.get('tables', {})
...
files = data.get('files', {})
readme = files.get('README')
```